### DISCUSSION TITLE
Public URL required in configuration?

### DISCUSSION THREAD

**Discussion by RadianM** [alandtse/alexa_media_player#2607] [Q&A] [answered]
I'm not sure why the configuration panel requests that we enter a **Public URL** to access Home Assistant? Not everyone exposes their LAN to the wider internet like me, and It seems to work OK with a local address... just curious about the choice of wording?

![Screenshot 2024-10-09 12 01 09](https://github.com/user-attachments/assets/4a763e53-7f77-4715-a2d5-c932195a537d)

**Comment by danielbrunt57** [comment, marked as answer]
It's only used in a few select cases and it does indeed need to be the publicly exposed URL. 
One such use case I heard about is when a third party, cloud-hosted notification service is used to somehow access a `notify.alexa_media` service (or something like that).  But for the rest of us, it's irrelevant and unused.
